source/docs/guides/merchant_report.md

```diff
@@ -0,0 +1,218 @@
+| `interac_online_enabled` | Does the merchant have interac online enabled |
+| `visa_src_enabled` | Does the merchant have visa checkout enabled |
+
+
+More detailed information about these fields can be found in our API
+specification which can be found at:
+<https://dev.na.bambora.com/docs/references/payment_APIs/v1-0-5/>
+
+## Getting Started
+
+The Merchant Report API is accessed via the URL
+<https://api.na.bambora.com/v1/reports/merchants>
+
+In addition, you will need use your Merchant ID and Reporting API Access
+Passcode from your Bambora account.  See the "Authentication" section below
+for more details on how to generate your access passcode to use in the request
+to the Merchant Report API.
+
+## API Requests
+
+### Request Headers
+
+Requests to the settlement endpoint should include two request headers:
+a `Content-type` header with the value `application/json` and a `Authorization`
+header which is outlined in the next section.
+
+### Authentication
+
+All requests to the Merchant Report require authentication in the form of an
+`Authorization` header.  This header contains the value `Passcode <YOUR API
+ACCESS PASSCODE>` which is an encoded version of your Merchant ID and your
+Reporting API Access Passcode.  Note that your Reporting API Access Passcode can
+be found in the [Portal](https://web.na.bambora.com) under
+**administration** > **account settings** > **order settings**.
+
+`<YOUR API ACCESS PASSCODE>` is created by concatenating your Merchant ID to
+your Reporting API Access Passcode (separated by a colon), and then base 64
+encoding the result.
+
+As an example, if your Merchant ID is 123456789, and your Reporting API Passcode
+is `fd36d245a3584434b904096525547f17`, then using Javascript your API Access
+Passcode could be generated by doing:
+
+```javascript
+merchantID = "123456789";
+reportingAPIPasscode = "fd36d245a3584434b904096525547f17";
+
+// base 64 encode the two values concatenated with a colon
+yourAPIAccessPassCode = btoa(merchantID + ":" + reportingAPIPasscode);
+```
+
+In this example `yourAPIAccessPassCode` would be
+`MTIzNDU2Nzg5OmZkMzZkMjQ1YTM1ODQ0MzRiOTA0MDk2NTI1NTQ3ZjE3` which is the value
+used when making a call to the Merchant Report API.
+
+Alternatively, for convenience, you can also use the
+[form](https://dev.na.bambora.com/docs/forms/encode_api_passcode/) which does
+same calculation.
+
+It is worth noting that while this process obfuscates/obscures your Reporting
+API Passcode it is not an *encrypted* value, so this value should be treated with
+equal sensitivity/security as your Reporting API Passcode itself.
+
+### Parameters
+
+The Merchant Report can accept one parameter to control the data returned by
+the endpoint: `merchants`, which is supplied as part of the URL
+
+If not passed the data returned will be for all the submerchants of the Channel Partner requesting the report.
+If a submerchant is passed in then only data for that submerchant will be returned
```

Comment:
I think submerchant should be sub-merchant to be consistent with other documentation